en/docs/modules/misc/KeyboardChip.md

```diff
@@ -152,6 +164,37 @@ Input names can be accessed by `KeyboardChip.`nameOfInput, and are listed below.
 
 ## Remarks
 
+### How to use
+The keyboard chip takes advantage of [CPU](.CPU.md) events, which can be hard to wrap your head around at first. The following is a brief tutorial on how to log the name of the key that's currently being pressed or released.
+
+First, place a keyboard chip and a CPU on your gadget and select the CPU with the Multitool:
+
+![Selecting the CPU](./../../../assets/docs/KeyboardChip/SelectingCPU.png)
+
+Then, select EventChannels and set the first channel to the keyboard chip.
+
+![Setting the event channel](../../../assets/docs/KeyboardChip/EventChannel.png)
+
+Now go into your CPU's code and add the following function:
+
+```lua
+function eventChannel1(sender:KeyboardChip, arg:KeyboardChipEvent)
+end
+```
+Remember that it MUST be global! Prepending the local keyword will prevent the CPU from recognizing it. Adding this kind of function will cause the CPU to automatically run it whenever the user presses or unpresses a key.
```

Comment:
Prefer markdown over capitals for legibility and semantics; _remember to not use unpress?_
```suggestion
Remember that it **must** be global! Prepending the `local` keyword will prevent the CPU from recognizing it. Adding this kind of function will cause the CPU to automatically run it whenever the user presses or releases a key.
```